clang/include/clang/Tooling/Core/Replacement.h

```diff
@@ -44,7 +44,8 @@ enum class ConstantFlagType : int {
   Default = 0,
   Host = 1,
   Device = 2,
-  HostDevice = 3
+  HostDevice = 3,
+  HostDeviceInOnePass = 4
```

Comment:
Please add comment for new added value:  what does in OnePass mean?  also highlight the differnece with value 3.